hhvm/content.md

```diff
@@ -0,0 +1,26 @@
+# What is HHVM?
+
+HHVM is an open-source virtual machine designed for executing programs written in Hack and PHP. HHVM uses a just-in-time (JIT) compilation approach to achieve superior performance while maintaining the development flexibility that PHP provides.
+
+> [wikipedia.org/wiki/HipHop_Virtual_Machine](https://en.wikipedia.org/wiki/HipHop_Virtual_Machine)
+
+![logo](https://raw.githubusercontent.com/baptistedonaux/docker-images/hhvm/logo.png)
+
+# How to use this image
+
+```console
+$ docker run --name hhvm -v /some/content:/usr/share/nginx/html:ro -P -d hhvm
```

Comment:
Since the default of the image is currently to run it with `fpm`, I think there should be some documentation on how to run something like the nginx container in front of it.  Some users will try to just point their browser at the fpm container and this would help alleviate those issues.

It should also mean that the `-P` could be dropped since the users will be hitting nginx rather than the hhvm container.